packages/mdc-list/_variables.scss

```diff
@@ -14,6 +14,5 @@
 
 $mdc-list-divider-color-light: rgba(0, 0, 0, .12) !default;
```

Comment:
The names of these variables are confusing - can we rename them while we're here? E.g.:

```sass
$mdc-list-divider-color-on-light-bg: rgba(0, 0, 0, .12) !default;
$mdc-list-divider-color-on-dark-bg: rgba(255, 255, 255, .2) !default;
```

We'll need to add `BREAKING CHANGE` to the PR description (and the commit message when we squash and merge).